device.go

```diff
@@ -398,7 +398,7 @@ func updateSpecDeviceList(device pb.Device, spec *pb.Spec, devIdx devIndex) erro
 	}
 
 	stat := syscall.Stat_t{}
-	if err := syscall.Stat(device.VmPath, &stat); err != nil {
+	if err := syscall.Stat(vmPath, &stat); err != nil {
 		return err
```

Comment:
I wonder if it would be useful to wrap this error in some context? Something like

```
err = grpcStatus.Errorf(codes.Internal, "Invalid device VM path %s: %s", vmPath, err)
```